frontend/app/src/App.tsx

```diff
@@ -378,7 +382,12 @@ export class App extends PureComponent<Props, State> {
       connectionStateChanged: this.handleConnectionStateChanged,
       claimHostAuthToken: this.hostCommunicationMgr.claimAuthToken,
       resetHostAuthToken: this.hostCommunicationMgr.resetAuthToken,
-      setAllowedOriginsResp: this.hostCommunicationMgr.setAllowedOriginsResp,
+      onHostConfigResp: (response: IHostConfigResponse) => {
```

Comment:
Good point 👍  I did another iteration today to apply a small naming update to add more clarity and also apply some deconstruction: The HostConfig is now split up into the `LibConfig` and `AppConfig`. The `LibConfig` contains all settings relevant to streamlit-lib and the `AppConfig` the settings relevant for streamlit-app. 
`allowedOrigins`, `useExternalAuthToken`, and `enableCustomParentMessages` would all go into `AppConfig` since they are not used or relevant for the streamlit-lib part.